### ISSUE TITLE
Paranoid is not included on the type 'Includeable'

### ISSUE THREAD

**Issue by mjdavidson** [sequelize/sequelize#10787]
## What are you doing?

*This is a typescript specific error*
```js
const user = sequelize.define('user', {id: DataTypes.UUID},{paranoid:true})
const project = sequelize.define('project', {id: DataTypes.UUID})
Project.hasMany(User)
User.belongsTo(Project)

const project = await Project.create()
const user1 = await User.create({projectId: project.id})
const user2 = await User.create({projectId: project.id})
await user1.destroy()

const findProject = await Project.findOne({include:[{model: User, paranoid:true}]})
// findProject = {id: 1, users:[{id:1,id:2}]}


```

## What do you expect to happen?
```js
findProject = {id: 1, users:[{id:1,id:2}]}
```

## What is actually happening?
type error: 
Type '{ model: typeof User; paranoid: boolean; }' is not assignable to type 'Includeable'.
  Object literal may only specify known properties, and 'paranoid' does not exist in type 'Includeable'.

__Dialect:__  any
__Dialect version:__ XXX
__Database version:__ XXX
__Sequelize version:__ 5.6
__Tested with latest release:__ yes

**Comment by SimonSchick** [comment]
PR is welcome.

**Comment by papb** [comment]
Fixed by #10804
